src/components/scripture/dto/scripture-reference.transformer.ts

```diff
@@ -0,0 +1,31 @@
+import { applyDecorators } from '@nestjs/common';
+import { Transform } from '../../../common/transform.decorator';
+import { books } from '../books';
+import { bookIndexFromName } from '../reference';
+import { ScriptureReferenceInput } from './scripture-reference.dto';
+
+export const ScriptureStart = () =>
+  applyDecorators(
```

Comment:
`applyDecorators` joins multiple decorators into a single one. Since we only have one here, this isn't needed.